source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -147,6 +158,16 @@ export class Carousel extends Container {
         return this._pages[index];
     }
 
+    addPage(page: CarouselPage) {
+        if (!page.parent) {
+            this._pages.push(page);
+
+            page.setParent(this);
+        } else {
+            throw new Error(Strings.errors.pageAlreadyBelongsToAnotherCarousel());
```

Comment:
why do we have this error? If I understand correctly, we only support one carousel.
what's the problem if we do reparent?

Reply:
Good point - I'll remove this conditional :)